chain/epoch_manager/src/test_utils.rs

```diff
@@ -74,23 +75,25 @@ pub fn epoch_info(
 }
 
 pub fn epoch_config(
-    epoch_length: BlockIndex,
-    num_shards: ShardId,
-    num_block_producers: ValidatorId,
-    num_hidden_validators: ValidatorId,
+    epoch_length: HeightDelta,
+    num_shards: NumShards,
+    num_block_producer_seats: NumSeats,
+    num_fishermen: NumFishermen,
```

Comment:
There's no concept of `num_fishermen`. This should be `num_hidden_validators` (as it was), and the type should be just `NumSeats`

Reply:
++